src/frontend/views/post.njk

```diff
@@ -44,4 +44,23 @@ limitations under the License.
             </div>
         </div>
     </div>
+    <div class="card-footer">
+        <div style="float:right">
+                    <div style="display: inline-block;">
```

Comment:
[ISSUE] Please reduce the indentation / reformat

Reply:
Reformatted